charts/trino/values.yaml

```diff
@@ -125,10 +125,26 @@ initContainers: {}
   #     image: busybox:1.28
   #     command: ['sh', '-c', 'echo The worker is running! && sleep 3600']
 
+sidecarContainers: {}
```

Comment:
We already document that `initContainers` can be used for sidecar containers: https://docs.starburst.io/latest/k8s/sep-configuration.html#sidecar-and-init-containers

Reply:
Does it apply to trino?